portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +10,19 @@
     <div id="content">
       <h1>Shengjing Zhang's Portfolio</h1>
       <p>Welcome to my portfolio</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      
+      <p>Click here to see a greeting message:</p>
+      <button onclick="getJsonMessage()">Fetch Json message!</button>
+      <div id="json-container"></div>
+
+      <h1 id="comment"></h1>
+      <form action="/data" method="POST">
+        <p>Enter your comment:</p>
+        <textarea name="comment-input"></textarea>
+        <br/><br/>
```

Comment:
Consider using CSS style instead of adding br tags as they might be hard to keep track of later.